From bb973fa33952c1a803097852c031b3979dae607c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Tue, 22 Dec 2020 09:38:13 +0100
Subject: [PATCH] Fix test_session with msgpack-1

Replace the encoding parameter with raw=False to fix test_session
with msgpack-1.0.0+.  The encoding parameter was already deprecated
in msgpack-0.6.2, and raw=False is compatible with that version too.
---
 jupyter_client/tests/test_session.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jupyter_client/tests/test_session.py b/jupyter_client/tests/test_session.py
index d0ec662..45be9a9 100644
--- a/jupyter_client/tests/test_session.py
+++ b/jupyter_client/tests/test_session.py
@@ -304,7 +304,7 @@ class TestSession(SessionTestCase):
 
         session = ss.Session(
             pack=msgpack.packb,
-            unpack=lambda buf: msgpack.unpackb(buf, encoding='utf8'),
+            unpack=lambda buf: msgpack.unpackb(buf, raw=False),
         )
         self._datetime_test(session)
 
-- 
2.29.2

